includes/ajax-handlers/class-wc-stripe-checkout-sessions-ajax-handler.php

```
@@ -125,10 +124,69 @@ public function create_checkout_session(): void {
+			$session_id = isset( $_POST['checkout_session_id'] )
+				? wc_clean( wp_unslash( $_POST['checkout_session_id'] ) )
+				: '';
+			if ( ! is_string( $session_id ) || '' === $session_id ) {
+				throw new Exception( __( 'Checkout session ID is required.', 'woocommerce-gateway-stripe' ) );
+			}
+
+			// Recalculate totals.
+			WC()->cart->calculate_totals();
+
+			$currency   = get_woocommerce_currency();
+			$cart_total = WC_Stripe_Helper::get_stripe_amount( (float) WC()->cart->get_total( 'edit' ), $currency );
```

PHPMD (2.15.0)

[error] Avoid using static access to class 'WC_Stripe_Helper' in method 'update_checkout_session'. (undefined)

(StaticAccess)